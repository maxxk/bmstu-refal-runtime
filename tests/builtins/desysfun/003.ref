$ENTRY Go
{
	= <PrintFunc 'I really need to study!!!' ('Ok!?!')><Open 'r' 1 'tmp.txt'><Prout <LOADFILE 1>>;
}

PrintFunc
{
	e.1 t.1 = <DeSysfun 'tmp.txt' (1000 Azaza 'Text' e.1 (Gena 'Bukin' 23) t.1)>;
}

$ENTRY LOADFILE 
{
	s.1 = <LOADFILE2 s.1 <Get s.1>>;
};

LOADFILE2 
{
    s.1 0 = ;
    s.1 e.2 0 = e.2;
    s.1 e.2 = e.2 '\n' <LOADFILE s.1>;
};
