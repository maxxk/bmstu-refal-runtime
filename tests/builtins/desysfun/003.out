Azaza 'TextI really need to study!!!'(Gena 'Bukin'23 )('Ok!?!')
